rct229/rules/section6/section6rule2.py

```diff
@@ -51,18 +54,24 @@ def get_calc_vals(self, context, data=None):
             space_u = context.user
 
             # get allowance
-            lighting_power_allowance_p = table_9_6_1_lookup(
-                space_p["lighting_space_type"]
-            )
+            if space_p["lighting_space_type"] in [
+                GUEST_ROOM,
+                DORMITORY_LIVING_QUARTERS,
+            ]:
+                lighting_power_allowance_p = table_9_6_1_lookup(
+                    space_p["lighting_space_type"]
+                )["lpd"]
+            else:
+                assert space_p["lighting_space_type"] == DWELLING_UNIT
```

Comment:
I don't think this `assert` is needed if the `Section21Rule6` class uses `list_filter`.
When using the `list_filter` it sets the precondition for the logic that the `lighting_space_type` shall only be `GUEST_ROOM`, `DORMITORY_LIVING_QUARTERS` and `DWELLING_UNIT`
In that case, we can just say:
```python
if space_p["lighting_space_type"] in [
                 GUEST_ROOM,
                 DORMITORY_LIVING_QUARTERS,
             ]:
                 lighting_power_allowance_p = table_9_6_1_lookup(
                     space_p["lighting_space_type"]
                 )["lpd"]
             else:
                 lighting_power_allowance_p = DWELLING_UNIT_MIN_LIGHTING_POWER_PER_AREA
```

Reply:
Fixed Great point!